patches/server/0728-Implement-Spawner-Potential-API.patch

```diff
@@ -0,0 +1,294 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: thamid-gamer <60953955+thamid-gamer@users.noreply.github.com>
+Date: Sat, 17 Jul 2021 20:31:25 -0400
+Subject: [PATCH] Implement Spawner Potential API
+
+
+diff --git a/src/main/java/net/minecraft/world/level/BaseSpawner.java b/src/main/java/net/minecraft/world/level/BaseSpawner.java
+index c601b8b12756682a4cb300be8ebed4319902c5b5..00ff06c27c4f04a094135d627515f400a0e3f857 100644
+--- a/src/main/java/net/minecraft/world/level/BaseSpawner.java
++++ b/src/main/java/net/minecraft/world/level/BaseSpawner.java
+@@ -361,6 +361,14 @@ public abstract class BaseSpawner {
+         this.nextSpawnData = mobspawnerdata;
+     }
+ 
++    // Paper start
```

Comment:
If this logic in this function is copied from somewhere else in `BaseSpawner`, make a note of where its taken from for future updaters of this patch.